script/circleci

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+case $CIRCLE_NODE_INDEX in
+  0 )
+    bundle exec rake test
```

Comment:
Maybe it would be good to print which database we are testing here. Right we don't have this information on the ci's `script/circleci` output